src/platform/src/Bridge/Anthropic/ModelClient.php

```diff
@@ -47,11 +47,7 @@ public function request(Model $model, array|string $payload, array $options = []
             $options['tool_choice'] = ['type' => 'auto'];
         }
 
-        if (
-            isset($options['beta_features'])
-            && \is_array($options['beta_features'])
-            && !empty($options['beta_features'])
-        ) {
+        if (isset($options['beta_features']) && \is_array($options['beta_features']) && !empty($options['beta_features'])) {
```

Comment:
```suggestion
        if (isset($options['beta_features']) && \is_array($options['beta_features']) && $options['beta_features']) {
```

speaking about code style :)

Reply:
Indeed, will check the empty stuff, just saw the linebreaks 😊